openweathermap-ios/openweathermap-ios/MapViewController.swift

```diff
@@ -48,6 +56,32 @@ extension MapViewController: MKMapViewDelegate {
         return renderer
     }
     
+    func mapView(_ mapView: MKMapView, regionDidChangeAnimated animated: Bool) {
```

Comment:
I'm not sure, but it belongs to MapKit delegate, doesn't it?

Reply:
Yes, it belongs.